jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -151,7 +151,10 @@ public void set(String image) {
     }
   }
 
-  /** Configuration for {@code container} parameter. */
+  /**
+   * Configuration for {@code container} parameter. Note that {@code entrypoint} and {@code args}
+   * are {@code @Nullable} do handle inheriting values from the base image.
```

Comment:
"do handle" --> "to handle"

I'd put the new comment as non-Javadoc comment as `// Note that ...` below the Javadoc; I think Javadoc usually for code consumers rather than holding internal implementation details.